### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by ericgaspar**
- Complete scripts rewrite

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

File: scripts/install [hunk 1]
```diff
@@ -10,198 +10,198 @@ source _common.sh
...
-cd ccl
-if [ $(grep -c "flags.* lm .*" /proc/cpuinfo) -eq 0 ]
-then
-    cp scripts/ccl /usr/bin/ccl
-else
-    cp scripts/ccl64 /usr/bin/ccl
-fi
-sed -e "s@CCL_DEFAULT_DIRECTORY=/usr/local/src/ccl@CCL_DEFAULT_DIRECTORY=/opt/ccl@" -i /usr/bin/ccl
+ynh_app_setting_set --app=$app --key=final_path --value=$final_path
+git clone https://github.com/turtl/server "$final_path" --quiet
```
Comment (alexAubin) [inline scripts/install]: Zblerg, I don't know why the new trend is to use git clones more and more, is there any reason for this ?

**Comment by yalh76** [comment]
!testme

File: scripts/install (hunk 1 above)
Comment (yalh76) [reply scripts/install]: changed to the usual app.src

**Pull request merged**